### ISSUE TITLE
Issue with GitHub Actions's Opening PR step

### ISSUE THREAD

**Issue by Thomas-Boi** [devicons/devicon#321]
Hello all,

I have discovered three bugs with our current build scripts. 

What I found:
1. The checkout action couldn't check out a branch from a different repo. This is fixable by specifying a repository argument.
![image](https://user-images.githubusercontent.com/43018778/95796631-406c3280-0ca2-11eb-8f62-f6090ad62d56.png)

2. The PR action couldn't open a PR into a branch from a different repo. This might not be fixable since we can't (and shouldn't) get every person's git token just to contribute to our repo. 
![image](https://user-images.githubusercontent.com/43018778/95796578-1a469280-0ca2-11eb-9da9-d514344a6376.png)

3. The CSS build script is not building the colored classes for aliases. This is fixable.

These bugs (except the last one) were found when merging PR #309 and PR #320 respectively. 

So far, I have addressed bug 1 and 3 in PR #320. PR #320 is pushed from my own forked repo to mimic how most contributors contribute to our repo. 

For bug 2, we would have to modify our open PR step in the workflow. Here are two possible solutions that I thought of:

* We would open a new PR to the develop branch instead of the contributor's branch. 
  * Pros:
    * Would work since our github bot has permission to open PR into the develop branch
    * Doesn't mess up the contributor's git history if something went wrong (no need to force push to fix anything etc...)
  * Cons:
    * This would create an extra branch for every PR => some extra work to delete them afterwards
* We create a new workflow that would upload the new icons to Icomoon just for checking. The script would take a snapshot so we know what the icon looks like. There will be no building here. Once we merge the icons into the repo, we can create a schedule task that will actually build the icons, fonts, and css.
  * Pros:
    * Reduce the time running the build script for every icon-request PR
    * Not too hard since we have the majority of the Selenium code to do this. We might have to refactor a few things though
  * Cons:
    * We will have to create a new workflow file => take more time before we can officially start using the new build script.

As always, let me know what you guys think.

**Comment by amacado** [comment]
I would suggest we start with proposed solution 1 "We would open a new PR to the develop branch instead of the contributor's branch" and after this works we can work on the more advanced solution 2 "We create a new workflow.." (which I would prefer since it's a cleaner solution imo)

**Comment by Thomas-Boi** [comment]
That sounds good for me. Solution #2 can be put in a backlog and we can implement that at a later date. I'll start working on solution 1 until then
